cogs/make_member.py

```diff
@@ -224,7 +226,7 @@ async def make_member(self, ctx: TeXBotApplicationContext, group_member_id: str)
         # NOTE: The "Member" role must be added to the user **before** the "Guest" role to ensure that the welcome message does not include the suggestion to purchase membership
         await interaction_member.add_roles(
             member_role,
-            reason="TeX Bot slash-command: \"/makemember\"",
+            reason='TeX Bot slash-command: "/makemember"',
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes